From 48341a8b0d146c586b248c7c1479974560969638 Mon Sep 17 00:00:00 2001
From: Qu Wenruo <wqu@suse.com>
Date: Sat, 4 Feb 2023 16:48:00 +0800
Subject: [PATCH 143/144] drivers: pci: controller: add PCIE controller driver
 for RK3588

According to the downstream code, the RK3568 and RK3588 share the same
driver.

Downstream code only has extra handling for the following cases:

- RK1808 driver
- Bifurcation handling
- Endpoint handling

All of the above features are not available upstream anyway.

Thus here we only need to add a new compatible string for the existing
driver.

Signed-off-by: Qu Wenruo <wqu@suse.com>
---
 drivers/pci/controller/dwc/pcie-dw-rockchip.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/controller/dwc/pcie-dw-rockchip.c b/drivers/pci/controller/dwc/pcie-dw-rockchip.c
index c1e7653e508e..435b717e5bc6 100644
--- a/drivers/pci/controller/dwc/pcie-dw-rockchip.c
+++ b/drivers/pci/controller/dwc/pcie-dw-rockchip.c
@@ -354,6 +354,7 @@ static int rockchip_pcie_probe(struct platform_device *pdev)
 
 static const struct of_device_id rockchip_pcie_of_match[] = {
 	{ .compatible = "rockchip,rk3568-pcie", },
+	{ .compatible = "rockchip,rk3588-pcie", },
 	{},
 };
 
-- 
2.34.1

